src/Datadog.Trace.ClrProfiler.Managed/Instrumentation.cs

```diff
@@ -88,6 +90,29 @@ public static void Initialize()
             {
                 // ignore
             }
+
+            // we only support Service Fabric Service Remoting instrumentation on .NET Core (including .NET 5+)
+            if (string.Equals(FrameworkDescription.Instance.Name, ".NET Core", StringComparison.OrdinalIgnoreCase) ||
```

Comment:
I will improve this API in a follow-up PR to add something like `bool FrameworkDescription.IsDotNetCore()` that caches the result.